dataactvalidator/scripts/load_cfda.py

```diff
@@ -0,0 +1,16 @@
+from ftplib import FTP
+
+ftp = FTP('ftp.cfda.gov')      # connect to host, default port
+ftp.login()                    # user anonymous, passwd anonymous@
+ftp.cwd('usaspending')
+
+data = []
+ftp.dir('-t', data.append)
+file_listing = data[0]
```

Comment:
Can you insert some comments here (and further down) indicating the importance of the certain indices that you're referencing? 

Also some comments around the different steps being taken to get the file in question would be helpful for someone coming into this file later on.